CHANGELOG.rst

```diff
@@ -11,6 +11,11 @@ Unreleased
   ``dependson`` is now a list instead of a filter iterator. This fixes issue #72 
   where dependencies were only loaded in the first bulk load.
 
+**Changed**
+  Changed psycopg2 bulkloader documentation to use copy_expert instead of copy_from.
```

Comment:
copy_expert and copy_from is should be surrounded by double backticks to highlight them as code like `dependson` right above it.